pillar.example

```diff
@@ -1,5 +1,5 @@
 elasticsearch:
-  major_version: 2
+  version: 2.4.2-1
```

Comment:
Yep... so if this gets merged, you'd pin a specific version in pillar. Such as

    elasticsearch:
        version: 5.2.2-1